javax-money/src/main/java/com/fasterxml/jackson/datatype/money/MoneyModule.java

```diff
@@ -0,0 +1,168 @@
+package com.fasterxml.jackson.datatype.money;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.module.SimpleDeserializers;
+import com.fasterxml.jackson.databind.module.SimpleSerializers;
+import org.apiguardian.api.API;
+import org.javamoney.moneta.FastMoney;
+import org.javamoney.moneta.Money;
+import org.javamoney.moneta.RoundedMoney;
+
+import javax.money.CurrencyUnit;
+import javax.money.MonetaryAmount;
+import javax.money.MonetaryOperator;
+import javax.money.MonetaryRounding;
+import javax.money.format.MonetaryFormats;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+import static org.apiguardian.api.API.Status.STABLE;
+
+
+@API(status = STABLE)
+public final class MoneyModule extends Module {
```

Comment:
Should we actually rename this as `JavaxMoneyModule`?

... also, as a side-note, is there need for "Jakarta Money module"? (wrt javax licensing resulting in "forking" of javax APIs)

Reply:
Let me check on this.